easybuild/easyconfigs/c/CrayCCE/CrayCCE-19.06.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'CrayToolchain'
+
+name = 'CrayCCE'
+version = '19.06'
+
+homepage = 'https://pubs.cray.com/discover'
+description = """Toolchain using Cray compiler wrapper, using PrgEnv-cray
+(PE release: June 2019).\n"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
```

Comment:
@ekouts Please change this to `toolchain = SYSTEM` (same in the other `Cray*eb` easyconfigs).

That's a new requirement for the upcoming EasyBuild v4.0 (see https://boegel-eb.readthedocs.io/en/eb4/System_toolchain.html)